### PR TITLE
Add profile field for full name in local language.

### PR DESCRIPTION
**Pull request by johngian**

### CODE REVIEW

**Comment by johngian** [comment]
There is an extra commit with some UI fixes. It's kind of a scrappy way to do this so I left as a seperate commit. Need to squash before merge.

**Comment by johngian** [comment]
@akatsoulas r?

**Comment by johngian** [comment]
@ppapadeas UI r?

File: mozillians/users/models.py [hunk 1]
```diff
@@ -140,6 +141,8 @@ class UserProfile(UserProfilePrivacyModel):
     user = models.OneToOneField(User)
     full_name = models.CharField(max_length=255, default='', blank=False,
                                  verbose_name=_lazy(u'Full Name'))
+    full_name_local = models.CharField(max_length=255, blank=True,
```
Comment (akatsoulas) [inline mozillians/users/models.py]: Let's also add `default=' '`

File: mozillians/users/models.py [hunk 2]
```diff
@@ -140,6 +141,8 @@ class UserProfile(UserProfilePrivacyModel):
...
+                                       verbose_name=_lazy(u'Full name in local language'))
```
Comment (akatsoulas) [inline mozillians/users/models.py]: Let's also add default=' '

**Comment by akatsoulas** [comment]
Code and functionality look good. Indeed the UI feels a bit strange but I don't have any better suggestion atm. r+ regarding the review. I am leaving it up to you whether you want to merge it as is or wait for feedback/suggestions on the UI.

**Comment by johngian** [comment]
@akatsoulas PR updated

**Pull request merged**